App/Lettuce/Sources/Lettuce/UI/MessagesView/ThreadPreview.swift

```diff
@@ -70,10 +67,12 @@ extension ThreadPreview: View {
 
                             }
                         }
-                        .padding(.trailing, 43)
+                        .padding(.trailing, 8)
```

Comment:
please use `.design(padding:` that is closest :)

Reply:
fixed